PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/RewardedVideoAdUnit.java

```diff
@@ -17,11 +17,12 @@
 package org.prebid.mobile;
 
 import androidx.annotation.NonNull;
+import org.prebid.mobile.units.configuration.AdUnitConfiguration;
 
 public class RewardedVideoAdUnit extends VideoBaseAdUnit {
 
     public RewardedVideoAdUnit(@NonNull String configId) {
-        super(configId, AdType.REWARDED_VIDEO);
```

Comment:
This change looks ambiguous since the `VAST` type is not the same as `REWARDED_VIDEO`. As far as I remember for the rewarded video we use the `isOptIn` flag in the AdUnitConfig.